### ISSUE TITLE
Access to training code, provide incentives

### ISSUE THREAD

**Issue by AmitMY** [commaai/commavq#11]
This approach seems very interesting for other use cases.
For example, for my use case I would like to VQ videos of sign language, not of driving.
If the training code would be available (including fast video decoding, data augmentation, and training strategy), one could replicate your models on a different domain.

Why would you care?
If I can replicate this on my domain, I have high incentives to make this repo more performant.

**Comment by YassineYousfi** [comment]
Training code is really nothing special and you can easily replicate this using:

1- fast video decoding: https://github.com/NVIDIA/VideoProcessingFramework
2- VQVAE training code: https://github.com/CompVis/taming-transformers
3- GPT training code: https://github.com/karpathy/nanoGPT

I don't think we want this repo to be a general purpose vq/gpt training code. Other repos are doing this quite well!

**Comment by hyz-xmaster** [comment]
@YassineYousfi Hi, I would like to reproduce the results shown in this repo. Could you please provide some essential training details, like the number of training iterations, learning rate and the total batch size etc? Thx
